git_sizer_test.go

```diff
@@ -22,9 +23,27 @@ import (
 	"github.com/github/git-sizer/sizes"
 )
 
+func sizerExe(t *testing.T) string {
+	t.Helper()
+
+	v := "bin/git-sizer"
+	switch runtime.GOOS {
+	case "windows":
+		v = `bin\git-sizer.exe`
```

Comment:
Maybe do the more generic `v = v.ReplaceAll('/', '\')`?

Reply:
AFAICT the key thing here isn't the path separator (I think both are accepted) but the `.exe` extension.

Reply:
Oh, right, you need the `.exe` suffix. Sorry for missing that.

Still, it would be more flexible to use `v = v.ReplaceAll('/', '\') + '.exe'`, right?